test/infrastructure/docker/docker/machine.go

```diff
@@ -430,3 +434,21 @@ func (m *Machine) machineImage(version *string) string {
 
 	return fmt.Sprintf("%s:%s", defaultImageName, versionString)
 }
+
+func logContainerDebugInfo(ctx context.Context, name string) {
+	log := ctrl.LoggerFrom(ctx)
+
+	cmd := exec.CommandContext(ctx, "docker", "inspect", name)
+	output, err := exec.CombinedOutputLines(cmd)
+	if err != nil {
+		log.Error(err, "Failed inspecting the worker machine container", "output", output)
+	}
+	log.Info("Inspected the worker machine container", "output", output)
```

Comment:
```suggestion
	log.Info("Inspected the machine container", "output", output)
```
nit: might be this happens on CP nodes as well